src/api/console_stringifier.py

```diff
@@ -33,16 +34,61 @@ def _modify_unit(self, unit: str) -> str:
         """
         Returns the modified unit.
         """
-        unit = (
-            unit.replace(r"\squared", "^2")
-            .replace(r"\cubed", "^3")
-            .replace("\\per\\", "/")
-            .replace(r"\per", "/")
-            .replace("\\", " ")
-            .strip()
-        )
-
-        if unit[0] == "/":
-            unit = f"1{unit}"
+
+        # Detect "\squared" etc.:
```

Comment:
Really nit-picky, I know: Would you mind removing the `:` at the end of your comments for consistent styling across the project? It's just my convention to omit them everywhere else.

Reply:
Done.